lib/util.js

```diff
@@ -2,34 +2,63 @@ const unicode = require('../lib/unicode')
 
 module.exports = {
     isSpaceSeparator (c) {
-        return typeof c === 'string' && unicode.Space_Separator.test(c)
+        switch (c) {
+            case 0x0020:
+            case 0x00A0:
```

Comment:
So the build is failing because these 2 cases aren't covered. They aren't hit because those characters are handled in `parse.js()` directly. I like having them there (well at least 0x0020) because we want a fast codepath for the space character. But I can't really remove them from here because then the function would become incorrect (wouldn't return `true` for some space characters).

How should I handle this? Add a line like below in the tests?
```js
        // Workaround to get full coverage
        t.strictSame(util.isSpaceSeparator(0x0020), true)
        t.strictSame(util.isSpaceSeparator(0x00A0), true)
```

Reply:
So, here's the funny thing: `isSpaceSeparator` never returned true for those characters *because* the more common cases were covered in `parse`. I'm okay with keeping those common characters out of `isSpaceSeparator` for now.

Reply:
For clarification, `unicode.Space_Separator` is a RegExp generated by including all Unicode Space Separator code points [excluding the more common ones](https://github.com/json5/json5/blob/a71e39392cc20c488197eaed538fe29d477850c1/build/unicode.js#L11). That's why `isSpaceSeparator` never return true for some characters, despite its name.

Reply:
Oh ok, I'll remove them from there then. We get full coverage and the function behavior doesn't change, problems solved.